keps/sig-docs/20191020-doc-policies-for-third-party-content.md

```diff
@@ -0,0 +1,339 @@
+consistent, fair decisions during the review process.
+
+## Motivation
+
+SIG Docs publishes Kubernetes
+documentation on kubernetes.io in line with its
+[charter](https://github.com/kubernetes/community/blob/master/sig-docs/charter.md#scope)
+and sets standards for website content. Prior to this KEP, there are no
+clear guidelines or standards for third-party and dual-sourced content.
+
+The Kubernetes documentation is currently a mix of both 1) documentation
+describing the Kubernetes open source project; and 2) content describing
+how to install or use Kubernetes on several third party Kubernetes
+offerings.
+
+Some third party content is necessary in order for Kubernetes to
+function. For example: container runtimes (containerd, CRI-o, Docker), 
+networking policy (CNI plugins), Ingress controllers, and logging all require 
+third party components. Pages like [Logging Using Elasticsearch and Kibana](https://kubernetes.io/docs/tasks/debug-application-cluster/logging-elasticsearch-kibana/) 
+are highly specific to a third party offering and seem more like third party 
+product documentation than Kubernetes open source documentation.
+
+### Goals
+
+The goal of Kubernetes documentation is to accurately document in-project
+functionality for Kubernetes, and to eliminate barriers to effective
+contribution and understanding.
+
+The goal of this KEP is to reach and document a consensus on what
+types of third-party content are appropriate for inclusion in Kubernetes
+documentation; standards for including third-party content; and to create
+consistent policies for docs handle third-party and dual-sourced content.
+
+To address its goal, this KEP focuses on the following issues:
+
+<del>
+
+1. What third party content is appropriate for inclusion in the Kubernetes
+documentation?
+
+Proposed: Third-party content is permitted if it's required for Kubernetes to
+function in-project.
+
+1. Does third party content in sections such as [Getting Started](https://kubernetes.io/docs/setup/)
+in the docs provide sufficient value to the reader that they should remain?
+
+Casual consensus says yes, with one modification:
+- Eliminate the [production environment table](https://kubernetes.io/docs/setup/#production-environment)
+with a link to [certified conformance partners](https://kubernetes.io/partners/#conformance).
+
+1. Is there a list of content pages that are so focused on third party product
+usage that they should be removed or updated from the Kubernetes documentation?
+
+See https://github.com/kubernetes/website/issues/15748.
+
+1. When should the Kubernetes documentation host third party content that isn't
+maintained by a Kubernetes SIG?
+
+As infrequently as possible, with linking preferred to hosting. 
+
+1. How does the Kubernetes project handle third party content that is not kept
+up to date or hosts?
+
+If content isn't refreshed within 180 days, notify stakeholders of 90 days to 
```

Comment:
Can we please explicitly say that we will do this "notify" step to all existing content that is on the chopping block?